Day-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -1,2 +1,32 @@
 import React from 'react';
 import './Button.css';
+function Button(props) {
```

Comment:
You're suppose to create one functional component and pass props down (i.e. numbers/operators/etc.). This defeat the purpose of reusability. I recommend to give it another attempt.